generators/entity-server/templates/src/main/java/package/web/rest/EntityResource.java.ejs

```diff
@@ -253,6 +253,11 @@ public class <%= entityClass %>Resource {
       <%_ } _%>
     <% } %>
   <%_ } _%>
+  <%_ if (requiresPersistableImplementation) { _%>
+
+      <%= entityInstance %>.setIsNew();
+
+  <%_ } _%>
```

Comment:
This will then also be called for normal save operations (creating new entity) and not just for updates - which would then produce an error as it will try to perform an UPDATE command instead...

Reply:
You won’t manage to access the persistable object from here using dto.
![0B519680-6B55-4161-A2B1-F6490036D945](https://user-images.githubusercontent.com/1417183/153392336-6d5308e8-4311-453c-a18e-b2ff3f3802d7.jpeg)

Reply:
Adjusted the suggestion